gapic-common/test/gapic/grpc/rpc_call/raise_test.rb

```diff
@@ -84,4 +84,37 @@ def test_wont_wrap_non_grpc_errors
     assert_kind_of Time, deadline_arg
     assert_equal 1, call_count
   end
+
+  ##
+  # Tests that if a layer underlying the RpcCall throws a ::GRPC::Unavailable
+  # that contains a Signet::AuthorizationError in its text,
+  # it gets extracted and rewrapped into a G
+  def test_will_rewrap_signet_errors
+    signet_error_text = <<-SIGNET
+    #<Signet::AuthorizationError: Authorization failed.  Server message:
+    # {
+    #   "error": "invalid_grant",
+    #   "error_description": "Bad Request"
+    # }
```

Comment:
Are you sure this is correct? The hashes seem odd.

Reply:
hashes?

Reply:
The hash symbols on lines 95-98. Are they part of the server message? They look like someone attempted to comment out those lines (which of course wouldn't work in the middle of a string.) Just wanted to make sure.

Reply:
It does not really matter for a unit test since we're testing the code that looks for `Signet::AuthorizationError` to reclassify

Reply:
I guess that's true. Then nit: get rid of the hash symbols so it doesn't look like someone tried to comment something out here.